src/core/load-fragment.c

```diff
@@ -6626,3 +6627,40 @@ int config_parse_protect_hostname(
         free_and_replace(c->private_hostname, h);
         return 1;
 }
+
+int config_parse_bpf_delegate(
```

Comment:
Please use `DEFINE_CONFIG_PARSE_PTR()` marco and define 4 conf parsers.

Reply:
And put 4 different parsers in load-fragment-gperf.gperf.in?
I thought that a single parser with a switch was more efficient.